suite-native/module-trading/src/components/general/TradeableAssetsSheet/TradeableAssetsSheetHeader.tsx

```
@@ -0,0 +1,40 @@
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+import { FOCUS_ANIMATION_DURATION, SearchableSheetHeader } from '../SearchableSheetHeader';
+import { TradeableAssetsFilterTabs } from './TradeableAssetsFilterTabs';
+
+type TradeableAssetsSheetHeaderProps = {
+    onClose: () => void;
+};
+
+const HEADER_HEIGHT = 160;
+
+const wrapperStyle = prepareNativeStyle<{}>(() => ({
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)